frontend/src/scenes/toolbar-launch/AuthorizedUrlsTable.tsx

```diff
@@ -107,9 +108,10 @@ export function AuthorizedUrlsTable({ pageKey, actionId }: AuthorizedUrlsTableIn
                                             <LemonButton
                                                 fullWidth
                                                 type="stealth"
+                                                compact
+                                                icon={<IconEdit />}
```

Comment:
TBH I propose removing `icon` and `compact` from this button and the delete one below, in order to make this match the pattern used in all other ellipsis menus.